.gitignore

```diff
@@ -22,3 +22,4 @@ coverage.xml
 .vs/
 decksite/images/*-cmc.png
 .web_cache/
+my_doodle.py
```

Comment:
If you have things (like the above file) that are only applicable to you, you should generally put them in `.git/info/exclude`, not `.gitignore`.